CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and and th
 
 ## [Unreleased]
 
+### Changed
```

Comment:
vllt. sollte das einen eigenen Tag bekommen, finde das unter Changed bisschen viel. 
Ggf. sowas wie "Chores" oder "Version Bumps"

Reply:
According to [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) only those types of changes exist:

* Added for new features.
* Changed for changes in existing functionality.
* Deprecated for soon-to-be removed features.
* Removed for now removed features.
* Fixed for any bug fixes.
* Security in case of vulnerabilities.

In the example `Changed` is used for version changes.